components/empty-state/empty-state-action-link.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement, nothing } from 'lit';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { linkStyles } from '../link/link.js';
+
+/**
+ * `d2l-empty-state-action-link` is an empty state action component that can be placed inside of the default slot of `empty-state-simple` or `empty-state-illustrated` to add a link action to the component.
+ */
+class EmptyStateActionLink extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: The action text to be used in the subtle button
+			 * @type {string}
+			 */
+			text: { type: String },
+			/**
+			 * REQUIRED: The action URL or URL fragment of the link
+			 * @type {string}
+			 */
+			href: { type: String },
+		};
+	}
+
+	static get styles() {
+		return [bodyCompactStyles, linkStyles];
+	}
+
+	constructor() {
+		super();
+		this._missingHrefErrorHasBeenThrown = false;
+		this._missingTextErrorHasBeenThrown = false;
+		this._validatingAttributesTimeout = null;
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		this._validateAttributes();
+	}
+
+	render() {
+		const actionLink = this.text && this.href
+			? html`
+				<a class="d2l-body-compact d2l-link" href=${this.href}>
+					${this.text}
+				</a>`
```

Comment:
Nit: I'd suggest putting this all on one line if it doesn't hurt readability too much, just to avoid the extra empty text nodes (and potential spacing impact they'd have):

```suggestion
				<a class="d2l-body-compact d2l-link" href=${this.href}>${this.text}</a>`
```